vertx-template-engines/vertx-web-templ-freemarker/src/main/java/io/vertx/ext/web/templ/freemarker/FreeMarkerTemplateEngine.java

```diff
@@ -28,15 +28,9 @@
  */
 @VertxGen
 public interface FreeMarkerTemplateEngine extends TemplateEngine {
-  /**
-   * Default max number of templates to cache
-   */
-  int DEFAULT_MAX_CACHE_SIZE = 10000;
-
   /**
    * Default template extension
    */
-  @Deprecated
```

Comment:
Yes and no. Due to the way engines work, some require template names with extensions, others are optional, while this isn't an issue when you have 1 cache per engine instance, it can get problematic when say that you deploy both `handlebars` and `freemarker` and a `handlebars` template is cached without extension (valid case) `template-file` and you cache a `freemarker` template which requires a full file name `template-file`. Now the file system templates:

* `template-file`
* `template-file.hbs`

Will override each other, so it's better if we revert this deprecation.